### PR TITLE
Removed telemetry deep dive from events

### PR DESCRIPTION
**Pull request by lzap**
Also adding two weeks later with the same YouTube link. I am assuming it can be moved there.

### CODE REVIEW

**Comment by lzap** [comment]
Updated to Mar 20, please squash as I am using github web editor.

File: _data/events.yml [hunk 1]
```diff
@@ -1250,21 +1250,14 @@
    brownfield deployments. As always, you're welcome to participate
    in the show, either via YouTube Live chat, or via our IRC channel
    (Freenode/#theforeman).
-- title: 'Foreman Deep Dives: Telemetry in Foreman (with Lukáš Zapletal)'
+- title: 'Foreman Deep Dives: Telemetry in Foreman (CANCELED)'
   type: 'Livestream'
   location: virtual
   start: 2018-03-14 14:00 GMT
```
Comment (orrabin) [inline _data/events.yml]: This needs to change to 2018-03-21 (next line as well)

File: _data/events.yml [hunk 2]
```diff
@@ -1250,21 +1250,14 @@
    brownfield deployments. As always, you're welcome to participate
    in the show, either via YouTube Live chat, or via our IRC channel
    (Freenode/#theforeman).
-- title: 'Foreman Deep Dives: Telemetry in Foreman (with Lukáš Zapletal)'
+- title: 'Foreman Deep Dives: Telemetry in Foreman (CANCELED)'
```
Comment (orrabin) [inline _data/events.yml]: You should put back your name :)

File: _data/events.yml [hunk 3]
```diff
@@ -1250,21 +1250,14 @@
...
   description: |
-    Rails internal telemetry was recently added into Foreman Core,
-    in this demo Lukáš Zapletal is going to configure and show how
-    to integrate Foreman with Prometheus. The next part will cover
-    all currently exported metrics, how to read them and utilize for
-    finding bottlenecks. In the last part, he is gonna show how to
-    add your own telemetry data into core or plugins. As always,
-    you're welcome to participate in the show, either via YouTube
-    Live chat, or via our IRC channel (Freenode/#theforeman).
+    Rails internal telemetry deep dive was canceled. Moved to Mar 20 2018.
```
Comment (orrabin) [inline _data/events.yml:1260]: 21 not 20.

File: _data/events.yml (hunk 2 above)
Comment (orrabin) [inline _data/events.yml:1253]: why not move this instead of canceling and adding a new one?

**Comment by orrabin** [comment]
@lzap thanks for the PR, the date was fixed instead of canceling and adding a new one.

**Pull request closed without merging**